kong/clustering/data_plane.lua

```diff
@@ -343,8 +378,6 @@ function _M:communicate(premature)
               -- count is guaranteed to not exceed 1
               config_semaphore:post()
             end
-
-            send_ping(c)
```

Comment:
why is this removed?

Reply:
Because it sends hash in wrong place. It sends old hash, and we want it to send new hash.

Reply:
ok, let's include this and also the ping_immediately change in commit message (or ideally in seperate commits, either works better for you)

Reply:
It is now on separate commit.